js/data/currencies.js

```diff
@@ -683,10 +683,17 @@ function getTranslatedCurrencies(lang = app.localSettings.standardizedTranslated
       ' should be returned in.');
   }
 
-  let translated = currencies.map((currency) => ({
-    ...currency,
-    name: app.polyglot.t(`currencies.${currency.code}`),
-  }));
+  let translated = currencies.map(currency => {
+    const name = app.polyglot.t(`currencies.${currency.code}`);
+    return {
+      ...currency,
+      name,
+      nameWithCode: app.polyglot.t('currencyWithCode', {
+        name,
+        code: currency.code,
+      }),
+    };
+  });
```

Comment:
Wouldn't it be simpler to leave currencies.js unchanged, and just call the currencyWithCode in the template, like this:
`<%= ob.polyT('currencyWithCode', { name: currency.name, code: currency.code }) %>`?

Reply:
It's just a question/suggestion, not a requirement.